app/controllers/api/mobile/petitions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::Mobile::PetitionsController < ActionController::Base
+
+  attr_reader :petition_cached_service
+
+  def initialize(petition_cached_service: PetitionCachedService.new)
```

Comment:
I'd rather call this `PetitionService` instead. Caching is an implementation detail. Later we could even receive options to force a fresh fetch, etc.